core/objects.go

```diff
@@ -125,7 +125,7 @@ type ValidationRecord struct {
 	URL string `json:"url,omitempty"`
 
 	// Shared
-	Hostname          string   `json:"hostname,omitempty"`
+	DnsName           string   `json:"hostname,omitempty"`
```

Comment:
Do we want to adjust the JSON field of `hostname` here? If so, there could be some downstream workflows that will need changes as well (e.g. log checkers).

Reply:
Yeah, it was my thinking that it's okay to leave the json name of this field as-is for now, since we're going to have to replace this field entirely in the near future anyway. No need for the downstream tools to have to go through two changes, when the main point of this change is to just get Boulder using the same language everywhere to make future changes easier.